ui/src/logic/beaconchaApi/index.ts

```diff
@@ -0,0 +1,74 @@
+import { BeaconchaGetResponse } from "./types";
+import { ApiParams } from "../../types";
+
+//TODO Use inheritance for both APIs?
+export class BeaconchaApi {
+  baseUrl: string;
+  endpoint: string;
+  fullUrl: string;
+
+  constructor(apiParams: ApiParams) {
+    this.endpoint = apiParams.apiPath;
+    this.baseUrl = apiParams.baseUrl;
+    this.fullUrl = this.baseUrl + this.endpoint;
+  }
+
+  /*
+   * Fetch info for every validator PK
+   */
+  public async fetchAllValidatorsInfo({
+    beaconchaApi,
+    allValidatorPKs,
+  }: {
+    beaconchaApi: BeaconchaApi;
+    allValidatorPKs: string[];
+  }): Promise<BeaconchaGetResponse[]> {
+    const validatorsInfo = new Array<BeaconchaGetResponse>();
+
+    allValidatorPKs.forEach(async (pubkey) => {
+      let validatorInfo = await beaconchaApi.fetchValidatorInfo(pubkey);
```

Comment:
Do not use variables of type let unless you are going to give another value in the future. Use `const`

Reply:
Done